sdk/communication/Azure.ResourceManager.Communication/samples/Sample1_ManagingCommunicationService.md

```diff
@@ -47,6 +47,34 @@ ArmOperation<CommunicationServiceResource> communicationServiceLro = await colle
 CommunicationServiceResource communicationService = communicationServiceLro.Value;
 ```
 
+***Create a Communication Service with User Assigned Managed Identity***
+
+```C# Snippet:Managing_CommunicationService_CreateAnApplication with User Assigned Managed Identity
+var communicationService = await communicationManagementClient.CommunicationServices.CreateAsync(
+    resourceGroupName,
```

Comment:
please declare the values of these variables just like the other examples. We want the clients to be able to copy and paste this and work out of the box